lib/template/places-json.php

```diff
@@ -0,0 +1,45 @@
+<?php
+	$aFilteredPlaces = array();
+
+	if (!sizeof($aPlaces))
+	{
+		if (isset($sError))
+			$aFilteredPlaces['error'] = $sError;
+		else
+			$aFilteredPlaces['error'] = 'Unable to geocode';
```

Comment:
I think it is better to return an empty list if none of the places could be found. (Same for XML response.) Is $sError even set anywhere?